### PR TITLE
Feature/KITOSUDV-4777

### PR DESCRIPTION
**Pull request by a-naskret**
## KITOS Pull request template
The following procedure dictates the steps needed before a Pull request can be merged into master.

- [x] **Implement**: 
      *All requirements are implemented and unit tests are green*
      
- [x] **Merge master into branch / rebase with master**: 
      *Make sure you are testing your changes and how they co-exist with the latest version of master*
      
- [x] **Green on integration**: 
      *All integration tests are green on integration*
      
- [x] **Request review**: 
      *Tag whomever you wish to review your code*
      
- [ ] **Review completed**: 
      *Reviewer ticks this box when review comments have been submitted*
      
- [ ] **Changes**: 
      *PR owner and reviewer agrees on which changes must be made and the changes are committed.*
      
- [ ] **Merge master into branch / rebase with master**: 
      *Make sure you are testing your changes and how they co-exist with the latest version of master*
      
- [ ] **Green on integration**: 
      *All integration tests are green on integration*

### CODE REVIEW

File: Core.ApplicationServices/System/ItSystemService.cs
Comment (strongmindsmhs) [inline Core.ApplicationServices/System/ItSystemService.cs:408]: What happens if newExternalUuid is null? - does this mean overwrite the old value or is it just a way of saying "no change"? After having created a new ItSystem and having set the externalUuid, how do we handle the situation where the other party no longer thinks there is an ItSystem?

File: Presentation.Web/Controllers/API/V2/External/ItSystems/Mapping/ItSystemWriteModelMapper.cs
Comment (strongmindsmhs) [inline Presentation.Web/Controllers/API/V2/External/ItSystems/Mapping/ItSystemWriteModelMapper.cs:1]: In general, do we violate separation of concerns (or SRP) by allowing for special DTO's concerning only RightsHolders? It's just an observation for now, but perhaps we need to look at the model for access controlling the itsystem fields .. maybe a more mixin approach or attributebased approach could be considered?

File: Presentation.Web/Models/API/V2/Request/System/RightsHolder/RightsHolderFullItSystemRequestDTO.cs
Comment (strongmindsmhs) [inline Presentation.Web/Models/API/V2/Request/System/RightsHolder/RightsHolderFullItSystemRequestDTO.cs:26]: Perhaps a comment stating the rules for ExternalUuid, eg wrt null

**Review by strongmindsmhs — APPROVED** [review]
Let's talk on possible changes

**Comment by a-naskret** [comment]
Can be merged after merging the FK Org fix

**Pull request merged**
